ui/packages/atlasmap/src/UI/ExpressionContent.tsx

```diff
@@ -421,6 +421,7 @@ export const ExpressionContent: FunctionComponent<IExpressionContentProps> = ({
     setTooltipVisible(true);
     tooltipTimeOut = setTimeout(() => {
       setTooltipVisible(false);
```

Comment:
What happens if the component gets removed before this callback gets fired?

Reply:
Then uninitializeMappingExpression() will get called on the unmount and cleartimeout() is called right?